### PR TITLE
Reduced chunk vendor build size ( Parsed size from 3.48MB to 1.19MB )

### PR DESCRIPTION
**Pull request by kamwing**
Hi, dear directus team

My name is kamwing and I'm from China. recently I use directus to build my project website. It's really easy to use and very flexible. Thank you very much for build this amazing project!

I found the Wysiwyg and markdown bundle size is too large, and not necessarily build into chunk-vendors, so I split out **Wysiwyg** and **markdown** code into a separate chunk. 

Before code split

After code split

### CODE REVIEW

**Comment by rijkvanzanten** [comment]
Hi kamwing! Nice to meet you 🙂 

> I found the Wysiwyg and markdown bundle size is too large, and not necessarily build into chunk-vendors, so I split out Wysiwyg and markdown code into a separate chunk.

Good point. As a matter of fact, all interfaces used to be Async components (in v7/v8), however this causes quite a lot of jumps on the page while the interfaces load. We realized it's a nicer UX to have a bigger load up front, and then a smooth operation while you're using the app. 

That being said, TinyMCE is very large, so I agree it makes sense to make an exception to that rule for that 👍🏻

File: app/src/interfaces/wysiwyg/index.ts [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 import { defineInterface } from '@/interfaces/define';
+const InterfaceWYSIWYG = () => import('./wysiwyg.vue' /* webpackChunkName: 'interface-wysiwyg' */);
 
 export default defineInterface(({ i18n }) => ({
 	id: 'wysiwyg',
 	name: i18n.t('interfaces.wysiwyg.wysiwyg'),
 	description: i18n.t('interfaces.wysiwyg.description'),
 	icon: 'format_quote',
-	component: InterfaceWYSIWYG,
+	component: InterfaceWYSIWYG as any,
```
Comment (rijkvanzanten) [inline app/src/interfaces/wysiwyg/index.ts:9]: Can we type this a little better? Maybe `as AsyncComponent` instead of `as any` ?

Comment (kamwing) [reply app/src/interfaces/wysiwyg/index.ts:9]: Hi @rijkvanzanten! Thanks for the reply.

I did some research from 
https://github.com/vuejs/vue-class-component/issues/323#issuecomment-624450239

```js
const AsyncComponent = () => ({
  component: import('.....'),
  loading: LoadingComponent,
}) as any;

@Component({
  components: {
    AsyncComponent,
    // or directly
    'async-xxx': () => ({
    	component: import('...'),
    	loading: LoadingComponent,
     }) as any,
  }
})
class App extends Vue {}
```

and

https://github.com/vuejs/vue/blob/2.6/types/vue.d.ts#L19

```
vue/types/vue.d.ts
```
```js
export interface CreateElement {
  (tag?: string | Component<any, any, any, any> | AsyncComponent<any, any, any, any> | (() => Component), children?: VNodeChildren): VNode;
  (tag?: string | Component<any, any, any, any> | AsyncComponent<any, any, any, any> | (() => Component), data?: VNodeData, children?: VNodeChildren): VNode;
}
```

this was the only way that works for me and maybe there is another way?

Comment (kamwing) [reply app/src/interfaces/wysiwyg/index.ts]: Hi @rijkvanzanten! Thanks for the reply.

I have changed  `as any` to `as AsyncComponent`.

> We realized it's a nicer UX to have a bigger load up front, and then a smooth operation while you're using the app.

I agree with this viewpoint. now I use webpack magic comment `/* webpackPrefetch: true */` to improve this UX.

截屏2021-03-24-下午11 47 52

**Review by rijkvanzanten — APPROVED** [review]

**Pull request merged**